lib/foreman_maintain/repos_manager/el.rb

```diff
@@ -0,0 +1,35 @@
+module ForemanMaintain::ReposManager
+  class El
+    include ForemanMaintain::Concerns::OsFacts
+    include ForemanMaintain::Concerns::SystemHelpers
+
+    def disable_repos(repo_ids)
+      if el8?
+        execute!("dnf config-manager --set-disabled #{repo_ids.join(',')}")
+      elsif el7?
+        execute!("yum-config-manager --disable #{repo_ids.join(',')}")
+      end
+    end
+
+    def enable_repos(repo_ids)
+      if el8?
+        execute!("dnf config-manager --set-enabled #{repo_ids.join(',')}")
+      elsif el7?
+        execute!("yum-config-manager --enable #{repo_ids.join(',')}")
+      end
+    end
+
+    def enabled_repos_hash
+      yum_cmd = 'yum repolist enabled -d 6 -e 0 2> /dev/null'\
```

Comment:
should this use `dnf` on el8+? right now there is an alias, so calling yum instead of dnf works, but at some point it might be gone, and it seems nicer to log that we called "dnf" on modern OSes :)

Reply:
Yes, miss from my side, shall update!